python/tvm/relay/op/dyn/transform.py

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=import-outside-toplevel
+"""Dynamic Transform operators."""
+
+from . import _make
+
+
+def reshape(data, newshape):
```

Comment:
I suggest in the frontend we only keep one `reshape` and inside the frontend you can decide whether to use `dyn.reshape` or `reshape`. So this means that we remove this file and instead update the `reshape` in the op/transform.py

Reply:
This is an interesting idea. It makes buy into the dynamic op namespace implicit, which reduces changes needed in the frontends later, but it also forces people to use the namespace earlier. @jroesch What do you think, is implicit or explicit better at this stage?

Reply:
@jroesch Could you comment on this?

Reply:
I think its okay to have a single interface in the Python code, at least I talked with Haichen about it offline. It seems less confusing for end users who want to build models, but will require that people understand the difference between the two ops in the backend (which is true either way).

Reply:
Okay, I'll rework the python API to merge them.

Reply:
I merged the two interfaces, let me know what you think.